cpp/dolfinx/fem/FiniteElement.cpp

```diff
@@ -359,12 +244,6 @@ std::string FiniteElement<T>::signature() const noexcept
 }
 //-----------------------------------------------------------------------------
 template <std::floating_point T>
-mesh::CellType FiniteElement<T>::cell_shape() const noexcept
-{
-  return _cell_shape;
-}
```

Comment:
_cell_shape was left uninitialised for most elements. I'm unsure whether removing it or making sure it is set is a better plan

Reply:
We can remove, and add it back later if required.